prin

```diff
@@ -159,7 +159,7 @@ elif [[ "${POSITIONAL_ARGV[0]}" =~ ^[a-zA-Z0-9]*$ ]]; then
 				|grep "$COMMIT" \
 				|grep "Merge pull request" \
 				|cut -d'-' -f1)
-    mapfile -i CP_COMMITS < <(git log --pretty=format:"%H - %an, %ar : %s" \
+   mapfile -t CP_COMMITS < <(git log --pretty=format:"%H - %an, %ar : %s" \
```

Comment:
Only nit would be this. If you could, could you align mapfile?

Reply:
@pswica , Sorry for this, I will take care of this next time. Thanks again for your review efforts here.

Reply:
@xauthulei 
Nothing to be sorry for it was a really minor nit. Thanks a lot for your good work here, and for spotting things that I fudged up :)